Changed dropdown styles

gmisail opened the pull request:
I felt that the style of the dropdown did not match the style of the site, so I changed it to have a style in line with the rest. Notably, I added the same shadow property as the other cards. I also fixed an issue where a highlighted button would fill the entire button element, ignoring the border-radius. Let me know if you hate it, we may have to play around with the colors to maximize contrast between it and the background, or just scrap it all together.

metzlr commented:
![localhost_3000_room_97b6f50d-031a-476e-8e0f-7043426d6aff](https://user-images.githubusercontent.com/54820894/98777997-85b97800-23a6-11eb-9c2b-2a78c26e43c5.png)

What do you think of this. I added icons to the different buttons, fixed the delete button not being red, and cleaned up some redundant styling.

gmisail commented:
I adjusted the sizing a little bit because I think it was a little bit too big, or at least there was quite a bit of dead space. In particular, I changed the line height and font size (0.9 to 0.8, so a really minuscule change)

![Screenshot_2020-11-11 React App](https://user-images.githubusercontent.com/8661089/98861652-a67bdf00-2433-11eb-96fc-93c7f9f1a374.png)

metzlr commented:
Looks good
